reframe/frontend/cli.py

```diff
@@ -581,10 +653,26 @@ def main():
     printer.debug(format_env(options.env_vars))
 
     # Setup the check loader
+    if options.retry_failed is not None:
+        if options.retry_failed:
+            filename = options.retry_failed
+        else:
+            filename = os_ext.expandvars(
```

Comment:
This is not tested either, because unit tests have passed and this is invalid. It should be `osext`.